File: App/Services/Microservices.Demo.Policies.Service/Framework/Rest/Models/Requests/CreatePolicyRequest.cs
using Microservices.Demo.Policies.Service.Framework.Rest.Models.Dtos;

namespace Microservices.Demo.Policies.Service.Framework.Rest.Models.Requests
{
    public class CreatePolicyRequest
    {
        public string OfferNumber { get; set; }
        public PersonDto PolicyHolder { get; set; }
        public AddressDto PolicyHolderAddress { get; set; }
    }
}

File: App/Clients/ng-microservices-demo-client/src/app/infrastructure/rest-clients/policies/models/requests/create-policy-request.ts
import { AutoMap } from "@automapper/classes";
import { PersonDto } from "../dtos/person-dto";
import { AddressDto } from "../dtos/adress-dto";
export class CreatePolicyRequest {
  @AutoMap()
  public offerNumber: string;
  @AutoMap()
  public policyHolder: PersonDto;
  @AutoMap()
  public policyHolderAddress: AddressDto;

  constructor(
    offerNumber: string = '',
    policyHolder: PersonDto = new PersonDto(),
    policyHolderAddress: AddressDto = new AddressDto()
  ) {
    this.offerNumber = offerNumber;
    this.policyHolder = policyHolder;
    this.policyHolderAddress = policyHolderAddress;
  }
}
